unified-scheduler-logic/src/lib.rs

```diff
@@ -1051,11 +1065,23 @@ mod tests {
 
     #[test]
     fn test_existing_blocking_task_then_newly_scheduled_task() {
-        let sanitized = simplest_transaction();
         let address_loader = &mut create_address_loader(None);
-        let task1 = SchedulingStateMachine::create_task(sanitized.clone(), 101, address_loader);
-        let task2 = SchedulingStateMachine::create_task(sanitized.clone(), 102, address_loader);
-        let task3 = SchedulingStateMachine::create_task(sanitized.clone(), 103, address_loader);
+        let conflicting_address = Pubkey::new_unique();
```

Comment:
Since the `RuntimeTransaction` does not implement `Clone`, we need to create the transactions separately here.

though I think this change makes the test more clear by explicitly labeling `conflicting_address` as a conflict.